src/main/java/fruitshop/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,22 @@
+package fruitshop.service.impl;
+
+import fruitshop.model.Storage;
+import fruitshop.service.ReportGenerator;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    public static final String HEADER = "fruit,quantity";
+    public static final String COMA = ",";
+    private final Storage storage = new Storage();
```

Comment:
The `ReportGeneratorImpl` class should not contain a static final `HEADER` or `COMA` as they are specific to the CSV format. If the report format changes in the future, you will need to modify this class. It's better to extract these constants to a utility class or configuration file, or pass them as parameters.